### PR TITLE
add walkthrough-locations attribute

### PR DESCRIPTION
**Pull request by finp**
To support 'Publishing' walkthrough, user needs to understand the current configuration before changing it. This PR allows us to include the current config in walkthrough

### CODE REVIEW

File: src/common/docsHelpers.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ const getUserAttrs = (walkthroughId, username) => ({
     walkthroughId || buildValidProjectNamespaceName(username, 'shared')
   ),
   'user-username': username,
-  'user-sanitized-username': cleanUsername(username)
+  'user-sanitized-username': cleanUsername(username),
+  'walkthrough-locations': process.env.WALKTHROUGH_LOCATIONS
```
Comment (finp) [inline src/common/docsHelpers.js]: @aidenkeating  this isn't working for me, any ideas?

Comment (aidenkeating) [reply src/common/docsHelpers.js]: @finp I believe `docsHelpers.js` is only used from the client-side? The browser wouldn't have access to the env var by default. We could make a change to pass it from the server to the client if we need it.

Comment (finp) [reply src/common/docsHelpers.js]: @pb82 seems like we need a new plan? But you mentioned that there's a plan to add default value to tutorial web app operator, right? That would also solve my issue in docs, if there's a jira?

Comment (pb82) [reply src/common/docsHelpers.js]: @finp ah right, sorry. I forgot that that code is only ever run on the client. Looking for that JIRA now.

**Comment by finp** [comment]
closing as useless :wink:

**Pull request closed without merging**